src/fvOptions/axialFlowTurbineALSource/axialFlowTurbineALSource.C

```diff
@@ -475,10 +475,14 @@ void Foam::fv::axialFlowTurbineALSource::calcEndEffects()
             {
                 vector elementVelDir = elementVel / mag(elementVel);
                 scalar relVelOpElementVel = -elementVelDir & relVel;
-                vector rotorPlaneDir = freeStreamDirection_;
+                vector rotorPlaneDir = axis_; // use axis_ instead of rotorPlaneDir_
```

Comment:
Will this properly account for CW/CCW rotation?

Reply:
The better explanation would be: it works only if the angle between freeStreamDirection_ and axis_ is within the range of -90...0...90 degrees.
I should have added this as a comment.
Also, // use axis_ instead of rotorPlaneDir_ is not correct. It should have been // use axis_ instead of freeStreamDirection_.
Should I cancel this commit and make a new one?